config/test.config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "queueLength" : 20,
+  "logLevel"    : 3,
+  "logFile"     : null,
+  "ripple" : {
+    "trace"                 : false,
```

Comment:
get rid of everything that is not needed for the tests.  probably just hbase and hbase-rest are needed for now